packages/manager/modules/server-sidebar/src/web.constants.js

```diff
@@ -359,6 +376,15 @@ export const WEB_ORDER_SIDEBAR_CONFIG = [
     regions: ['EU', 'CA'],
     app: [WEB],
   },
+  {
+    id: 'orderWebPaas',
+    title: 'web_paas',
+    feature: 'web-paas',
+    icon: 'oui-icon oui-icon-partner-platformsh_concept',
+    state: 'web-paas.add',
+    regions: ['EU', 'CA', 'US'],
```

Comment:
```suggestion
    regions: ['EU],
```
only available in EU right?